program/image-classification-tf-py/classify.py

```diff
@@ -10,9 +10,17 @@
 import time
 import json
 import os
-import tensorflow as tf
+import sys
 import numpy as np
 
+# We have to augment search paths manually before importing TF
+# otherwise standard paths prepend ones from PYTHONPATH in `sys.path`
+# and pip-installed TF is loaded instead of ck-installed one
+for p in os.getenv('PYTHONPATH', '').split(':'):
```

Comment:
One niggle is that the path separator `:` is Linux-specific (it's `;` in Windows). Could you please try using `os.pathsep` here?